bower.json

```diff
@@ -16,11 +16,14 @@
     "output"
   ],
   "dependencies": {
-    "purescript-dom": "^3.1.0",
-    "purescript-aff": "^2.0.1",
-    "purescript-arraybuffer-types": "^0.2.0"
+    "purescript-dom": "^4.0.0",
+    "purescript-aff": "^3.0.0",
+    "purescript-arraybuffer-types": "^2.0.0"
   },
   "devDependencies": {
-    "purescript-console": "^2.0.0"
+    "purescript-console": "^3.0.0"
+  },
+  "resolutions": {
+    "purescript-arraybuffer-types": "^2.0.0"
```

Comment:
Yeah, maybe it's better to wait, if this gets merged it won't be necessary:

https://github.com/purescript-contrib/purescript-dom/pull/96/commits/28cbd24757301db60d966279667bbaf13c92b338